ifacemonitor/ifacemonitor_test.go

```diff
@@ -347,6 +350,15 @@ var _ = Describe("ifacemonitor", func() {
 		<-nl.userSubscribed
 	})
 
+	It("should skip netlink address updates for ipvs", func() {
+		SetDefaultConsistentlyDuration(1 * time.Second)
+		nl.addLink("kube-ipvs0")
+		resyncC <- time.Time{}
+		dp.expectAddrStateCb("kube-ipvs0", "", true)
```

Comment:
Two points here:

1. I'm not sure I understand the logic for *which* callbacks are skipped, for an excluded interface.  Specifically (a) is it correct that there is no LinkStateCb? - what bits of Felix are influenced by that?  (b) why *do* we get the initial AddrStateCb as shown above?  (It may be that this is all correct, but then it would be good to write a comment to explain it.)

2. What does `"", true` mean?  expectAddrStateCb doesn't have a precise expectation for this combination.